bin/pcov

```diff
@@ -71,16 +71,19 @@ if ($config["path"]) {
 	}
 }
 
-if (!\class_exists(\PHPUnit\Runner\Version::class)) {
-	usage("PHPUnit does not seem to be available");
+if (class_exists(PHPUnit_Runner_Version::class) &&
```

Comment:
I'm not sure why this is the first branch now ?

Reply:
Here https://github.com/krakjoe/pcov-clobber/pull/3/files#diff-81e693f3e3674a31805112e70f3c3843R77 you are testing for the existence for class `\PHPUnit\Runner\Version` - which exists only after PHPUNIT 6 or above.

If this class doesn't exists, the script will fail.

However in PHPUnit 5 the equivalent of this class is called `PHPUnit_Runner_Version`. I wanted to be sure that this class exists in conjunction with the right version number to determine the version, before we check on PHPUnit 6+ class syntax below, with minimal refactoring.

Reply:
Fair enough, I should have read it more closely :)